tests/functional/generate_pages.py

```diff
@@ -392,11 +392,13 @@ def process_calculated_summary(answers, page_spec):
         page_spec.write(CALCULATED_SUMMARY_LABEL_GETTER.substitute(answer_context))
 
 
-def process_list_collector_summary(schema_data, page_spec):
+def process_summary(schema_data, page_spec, collapsible):
     for section in schema_data["sections"]:
-        list_collector_blocks = QuestionnaireSchema.get_visible_list_blocks_for_section(
-            section
-        )
+        list_collector_blocks = [
+            summary_element
+            for summary_element in section.get("summary", [])
+            if summary_element["type"] == "List"
+        ]
```

Comment:
These aren't list collector blocks, so something like `list_summaries` would be a better name.